src/test/scala/io/iohk/ethereum/network/PeerEventBusActorSpec.scala

```diff
@@ -0,0 +1,144 @@
+package io.iohk.ethereum.network
+
+import java.net.InetSocketAddress
+
+import akka.actor.ActorSystem
+import akka.testkit.TestProbe
+import io.iohk.ethereum.Fixtures
+import io.iohk.ethereum.network.PeerEventBusActor.PeerEvent.{MessageFromPeer, MessageToPeer, PeerDisconnected, PeerHandshakeSuccessful}
+import io.iohk.ethereum.network.PeerEventBusActor.PeerSelector
+import io.iohk.ethereum.network.PeerEventBusActor.SubscriptionClassifier._
+import io.iohk.ethereum.network.PeersInfoHolderActor.PeerInfo
+import io.iohk.ethereum.network.p2p.messages.CommonMessages.Status
+import io.iohk.ethereum.network.p2p.messages.Versions
+import io.iohk.ethereum.network.p2p.messages.WireProtocol.{Ping, Pong}
+import org.scalatest.{FlatSpec, Matchers}
+
+class PeerEventBusActorSpec extends FlatSpec with Matchers {
+
```

Comment:
Do we have a test for multiple subscriptions to the same code resulting in a single event raised?